packages/common/__tests__/cors.test.ts

```diff
@@ -0,0 +1,82 @@
+import { Request } from 'cross-undici-fetch'
+import { request } from 'http'
+import { createServer } from '../src/server'
+
+describe('CORS', () => {
+  describe('Default behavior', () => {
+    const yoga = createServer()
+    it('should return the wildcard if no origin is sent with header', () => {
```

Comment:
I'm not sure this should be the default behaviour. Isn't having `*` by default considered insecure?

Reply:
See the next one. It returns wildcard if origin header is not present

Reply:
Yes but I mean we shouldn’t send wildcard if the origin isn’t specified. Should be an error, no?

Reply:
I.e default behaviour should be no CORS

Reply:
I don't think it should be an error unless the user limits the origins. Error would mean that origin is permitted and it would be weird an unknown origin is permitted even if user doesn't limit it.